test/wasi/c/stat.c

```diff
@@ -0,0 +1,53 @@
+#include <sys/stat.h>
+
+#include <assert.h>
+#include <fcntl.h>
+#include <time.h>
+#include <unistd.h>
+
+#define BASE_DIR "/tmp"
+#define OUTPUT_DIR BASE_DIR "/testdir"
+#define PATH OUTPUT_DIR "/output.txt"
+#define SIZE 500
+
+int main(void) {
+  struct stat st;
+  int fd;
+  int ret;
+  off_t pos;
+
+  (void)st;
```

Comment:
Any reason for this line?

Reply:
your guess is as good as mine, it's copied from lucet's tests